improver/cli/compare.py

```diff
@@ -51,10 +53,18 @@ def process(
         desired: path to desired/known good data netcdf file
         rtol: relative tolerance for data in variables
         atol: absolute tolerance for data in variables
+        ignored_attributes: list of attributes to ignore in the comparison
```

Comment:
Would it help to have a context note for this, as to which attributes / types of attributes are appropriate to ignore?

Reply:
I've added a slightly extended comment which includes an example of an attribute (history) that might wish to be ignored.